docker-compose.yaml

```diff
@@ -5,25 +5,27 @@ services:
       context: .
       dockerfile: Dockerfile
     environment:
```

Comment:
question; what inspired these changes?  Are you using `docker-compose run` in your personal workflow, or is this just an automated linter-cleaner-upper?

Reply:
Since this service has the option of operating without external dependencies I was fixing the `make run` flow so it could be used as a demo or as a tool security engineers could run on a laptop.